test/functional/integreatly_test.go

```diff
@@ -19,16 +19,19 @@ func TestIntegreatly(t *testing.T) {
 	t.Run("Integreatly Happy Path Tests", func(t *testing.T) {
 		for _, test := range common.ALL_TESTS {
 			t.Run(test.Description, func(t *testing.T) {
+				testingContext, err = common.NewTestingContext(config)
```

Comment:
i think we should be handling the `err` in these. 

Also I think we can remove from line 15 - 18,  we do not need to create the testing context here as we do it inline